lib/server/sthServer.js

```diff
@@ -227,19 +232,39 @@ function getKPIs() {
     };
 }
 
+/**
+ * Returns the server PIs
```

Comment:
What is PIs here?

Reply:
It's just a measurement value to return the number of processed requests. Named it along the lines of KPIs which returns the number of attended requests. Open to your suggestions for renaming it.

Reply:
getPI() -> getProcessedRequestWithErrorCount()
resetPI() -> resetProcessedRequestWithErrorCount()

A bit long, but I think that very descriptive. What do you think?

(In fact, I don't like too much getKPI() and resetKPI() for the other counter, but we can leave it as it is now)

Reply:
I think it's a good suggestion. Just a minor change I'd like to have.
getProcessedRequestWithErrorCount() -> getProcessedRequestsWithErrorCount()
resetProcessedRequestWithErrorCount -> resetProcessedRequestsWithErrorCount()

Reply:
Ah, yes... I missed the 's' in 'Request**s**'.

Yes, of course :)

Reply:
Fixed in c79c446